swaggen/src/main/java/mojo/AnnotationReaderMojo.java

```diff
@@ -121,26 +134,28 @@ public void execute() throws MojoExecutionException {
      * @throws MalformedURLException                 if URLs in the classpath are
      *                                               malformed
      */
-    private ClassLoader getClassLoader() throws DependencyResolutionRequiredException, MalformedURLException {
+    private ClassLoader getClassLoader(boolean showLogs) throws DependencyResolutionRequiredException, MalformedURLException {
         Set<URL> urls = new HashSet<>();
         List<String> elements = project.getRuntimeClasspathElements();
         for (String element : elements) {
             urls.add(new File(element).toURI().toURL());
+            if (showLogs) {
```

Comment:
@Billzzhang can we create a function that will check the value of showLogs and if it is set to true then it displays it instead of checking everywhere in the code.